src/llvm-final-gc-lowering.cpp

```diff
@@ -282,13 +283,6 @@ bool FinalLowerGC::runOnFunction(Function &F)
     LLVM_DEBUG(dbgs() << "FINAL GC LOWERING: Processing function " << F.getName() << "\n");
     // Check availability of functions again since they might have been deleted.
     initFunctions(*F.getParent());
-    if (!ptls_getter)
-        return true;
-
-    // Look for a call to 'julia.ptls_states'.
-    ptlsStates = getPtls(F);
-    if (!ptlsStates)
-        return true;
```

Comment:
Were these early returns some kind of optimization? If so, we can add something like

```cpp
if (!usePtls(F))
    return true;
```